src/dataflow/src/sink/kafka.rs

```diff
@@ -331,7 +331,15 @@ where
                     connector.frontier.less_equal(&time)
                 };
 
-                if !should_emit {
+                let previously_published = match &connector.consistency {
+                    Some(consistency) => match consistency.gate_ts {
+                        Some(ts) => time <= ts,
```

Comment:
I think we should use the `less_equal` method from line 331?

Reply:
Thanks! Change made